doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+
+The ParaView GUI is enabled by clicking on `Visualization` and checking the box for `Visualization GUI`. The `Visualization GUI` appears in the right-hand side of the NEAMS Workbench GUI with ParaView options as seen in [fig:hpc_on_demand_visualization_gui]. After enabling all variables and clicking `Apply` within the `Visualization GUI`, a variable can be plotted by right-clicking on the geometry in the *visualization tab* and selecting `Color by` and then the name of the variable of interest ( [fig:hpc_on_demand_color_by] and [fig:hpc_on_demand_color_by_power_density] ). The `Visualization tab` can also be split [to plot multiple variables side-by-side](https://docs.paraview.org/en/latest/UsersGuide/displayingData.html), as seen in [fig:neams_workbench_sfr_vis].
+
+!media neams_workbench/hpc_on_demand_visualization_gui.png style=width:80% id=fig:hpc_on_demand_visualization_gui caption=Enabling the Visualization GUI for ParaView.
+
+!media neams_workbench/hpc_on_demand_color_by.png style=width:80% id=fig:hpc_on_demand_color_by caption=After right-clicking on the geometry in the `Visualization tab`, select `Color by` to access all variables to plot.
+
+!media neams_workbench/hpc_on_demand_color_by_power_density.png style=width:80% id=fig:hpc_on_demand_color_by_power_density caption=NEAMS Workbench GUI showing `power density` profile in the *Visualization tab*.
+
+!media neams_workbench/neams_workbench_sfr_vis.png style=width:80% id=fig:neams_workbench_sfr_vis caption=The NEAMS Workbench GUI with *griffin.i* input file (top left), the output from the scheduler (bottom left), and the numerical solution visualized with ParaView (right)..
+
+### Exiting and closing the Virtual Desktop
+
+The Virtual Desktop can be exited by closing the browser tab that was opened when creating a `NEAMS Workbench session` in Step 1 (section 4.2.1). Users have the option of logging back in by clicking `Launch the NEAMS Workbench` shown in [fig:hpc_on_demand_open_session]. The NEAMS Workbench session can be closed by clicking on `Delete`, which will terminate the session.
+
+# Considerations
+
+Potential users should remember that this workflow is still under active development and that some VTB examples are not fully supported. For instance, the molten salt fast reactor VTB example relies on Nek5000-v19 to inform the Pronghorn model. Version 19 of Nek5000 execution is not currently integrated with the NEAMS Workbench; only version 17 execution is supported.
+
+Users may also see validation error messages when opening some of the VTB input files with the NEAMS Workbench, as shown at the bottom of the *griffin.i* input file in [Figure 9](#fig:hpc_on_demand_open_griffin). This is expected because some of the advanced MOOSE syntax requires a language server protocol to be fully cross-referenced and validated across multiple input blocks and input files. This work is in progress and will be available in the next releases of the NEAMS Workbench.
```

Comment:
```suggestion
Users may also see validation error messages when opening some of the VTB input files with the NEAMS Workbench, as shown at the bottom of the *griffin.i* input file in [Figure 9](#fig:hpc_on_demand_open_griffin). This is expected because some of the advanced MOOSE syntax requires a language server protocol to be fully cross-referenced and validated across multiple input blocks and input files. This work is in progress and will be available in future releases of the NEAMS Workbench.
```